modules/examples/src/main/scala/examples/CustomTheme.scala

```diff
@@ -0,0 +1,37 @@
+package examples
+
+import cats.effect.{IO, IOApp}
+import org.legogroup.woof.{*, given}
+import ColorPrinter.Theme
+import ColorPrinter.Theme.*
+
+object CustomTheme extends IOApp.Simple:
+
+  given Filter  = Filter.everything
+  given Printer = ColorPrinter(customTheme)
+
+  private def customTheme: Theme = Theme(
+    {
+      case LogLevel.Info  => Foreground.Black.withBackground(Background.Green)
+      case LogLevel.Warn  => Foreground.Black.withBackground(Background.Yellow)
+      case LogLevel.Trace => Foreground.Black.withBackground(Background.White).withStyle(Style.Underlined)
+      case LogLevel.Error => Foreground.Black.withBackground(Background.Red).withStyle(Style.Bold)
+      case level          => defaultTheme.levelFormat(level)
+    },
+    Foreground.Cyan.withStyle(Style.Underlined),
+    Style.Reset,
+    Empty,
```

Comment:
could we use named arguments for everything in this one, to make it clear what the "Empty" ones are supposed to be?